site/hugo.toml

```diff
@@ -116,7 +116,7 @@ copyright = "The InftyAI Team"
 #  Set to true to disable breadcrumb navigation.
 breadcrumb_disable = false
 # Set to false if you don't want to display a logo (/assets/icons/logo.svg) in the top navbar
-navbar_logo = true
+navbar_logo = false
```

Comment:
I just disabled the logo because the default logo path is (/assets/icons/logo.svg), however, we need to change to (/assets/icons/logo-white.svg), can we configure this? @rudeigerc

Reply:
ticketed an issue here: https://github.com/InftyAI/llmaz/issues/411

Reply:
I'll take a look at it.